index.html

```diff
@@ -33,6 +33,10 @@
                 <span>FPS: </span>
                 <span id='fps-status'>&nbsp;</span>
             </div>
+            <div class='info-block'>
+                <span>Hunter speed: </span>
+                <input id="hunter-speed-slider" type="range" min="0" max="100" value="15"/>

```

Comment:
Shouldn't the value here be 20? 
I'm not sure I'm reading it all right, but my understanding is that min and max can range from 0 to 100 here, however in the config they range from 1 to 1.25 and 20 to 25 respectively, with a starting values of 4 and 5. Therefore I think the value _here_ should match the percentage from 0 to 100 that 4 (and 5) is between 1 and 20 (and 1.25 and 25). I _think_ that means the value here should be 20. 

*Disclaimer, this was hard to think through, and I'm not confident I've got it all correct. Please double check. 

If it would be easier, you could try to assess this in debugging, but moving the slider away from 15, and then back to 15. What values for min and max does that set?